static/src/js/actions/urlQuery.js

```diff
@@ -170,7 +171,69 @@ export const changePath = (path = '') => async (dispatch) => {
       }))
     }
   } else {
-    decodedParams = decodeUrlParams(queryString)
+    const queryParams = parse(queryString)
+    const isNlpSearch = queryParams.nlp === 'true'
+
+    if (isNlpSearch && queryParams.q) {
+      useEdscStore.getState().query.changeQuery({
+        collection: {
+          keyword: queryParams.q
+        },
+        skipCollectionSearch: true
+      })
+
+      try {
+        const nlpRequest = new NlpSearchRequest(undefined, earthdataEnvironment)
```

Comment:
Redux is not the place to make the call to CMR, it is in a zustand function in the collections slice